docs/collections/_blueprints/general-ra-meta-tags.md

```diff
@@ -17,39 +17,48 @@ toc_label: "Table of Contents"
 gallery:
   - image: "/assets/images/blueprints/general-ra-meta-tags/ad03.png"
     caption: "<strong>Above:</strong> Application View"
-gallery_limit: 1
+  - image: "/assets/images/blueprints/general-ra-meta-tags/ad04.png"
+    caption: "<strong>Above:</strong> Reference Table View"
+gallery_limit: 2
 
 files:
   - title: "Application"
-    link: "https://github.com/XMPro/Blueprints-Accelerators-Patterns/blob/master/blueprints/general-ra-meta-tags/application/"
+    link: "https://github.com/XMPro/Blueprints-Accelerators-Patterns/blob/master/blueprints/general-ra-meta-tags/application"
     image: "/assets/images/icons/AD.png"
   - title: "SQL"
-    link: "https://github.com/XMPro/Blueprints-Accelerators-Patterns/blob/master/blueprints/general-ra-meta-tags/sql/"
+    link: "https://github.com/XMPro/Blueprints-Accelerators-Patterns/blob/master/blueprints/general-ra-meta-tags/sql"
     image: "/assets/images/icons/sql.webp"
+  - title: "Datastream"
+    link: "https://github.com/XMPro/Blueprints-Accelerators-Patterns/blob/master/blueprints/general-ra-meta-tags/datastream"
+    image: "/assets/images/icons/DS.png"
 
 ---
 
 ## Application
-This application allows the user to assign additional labels to a recommendation outside of the native recommendation editor after the recommendation and alerts have been created to allow for enhanced data analysis techniques.  The application is configured as follows: 
+
+This blueprint app establishes meta tag labels for recommendations. This allows historical recommendations to be filtered and analysed by labels such as asset class, location, training recommendations, or area. Intended uses could include:
+
+- Recommendation Analytics;
+- Filters available for 'Suspend Recommendations for shutdown';
+- Threshold comparisons by comparing various recommendations by asset class.
+
 
 ### Create Meta Tags
 
 <div class="inline_image">{% include framework/shortcodes/image.html src="/assets/images/blueprints/general-ra-meta-tags/ad01.png" %}</div>
-This page contains a navigation bar at the top right, a recommendation selection grid on the left, and create/ update card on the right. 
+This page contains a navigation bar at the top right and a recommendation selection grid to create/ update the meta tag. 
 
-Each grid has the Store User option enabled meaning it will remember your selection for column ordering, grouping and for the bottom grid hiding certain columns.  Selecting any of the rows in either grid will drilldown to the action page for the RCA that was selected.
+Each grid has the Store User option enabled meaning it will remember your selection for column ordering, grouping, and filtering.  
```

Comment:
```suggestion
Each grid has the Store User option enabled, which means it will remember your selections for column ordering, grouping, and filtering.  
```

Reply:
done